noah/src/main/resources/crawler/health_map_crawler.py

```diff
@@ -0,0 +1,95 @@
+import urllib2
+import urllib
+import datetime
+import io
+import os
+import json
+import sys
+ 
+
+#get health map response object
+def get_health_map(url, headers):
+    request = urllib2.Request(url, headers = headers)
+    response = urllib2.urlopen(request)
+    return response
+
+# create the parameters, needs start_date and end_date in "mm/dd/yyyy" format
+def generate_health_map_query(start_date, end_date):
+    parameter_map = {"category":["1","2","29"],"diseases":["282"], "heatscore":"1", "partner":"hm"}
+    parameter_map['sdate'] = start_date
+    parameter_map['edate'] = end_date
+    return parameter_map
+    
+    
+def getUrl(alertid, trto, trfr, pid):
```

Comment:
Lol. Ok. They are like that on the website. I don't know what they mean exactly. They are query parameters. I will try to give them meaningful names.